routers/repo/issue.go

```diff
@@ -48,7 +48,9 @@ func MustEnableIssues(ctx *middleware.Context) {
 }
 
 func MustEnablePulls(ctx *middleware.Context) {
-	if !ctx.Repo.Repository.EnablePulls {
+	if !ctx.Repo.Repository.CanEnablePulls() {
```

Comment:
@Unknwon That is not the case, though. Present mirror repositories will have EnablePulls set to true (as it is the default value), while mirrors should not support Pull Requests as of this commit.